templates/patterns/modal/modal.ui_patterns.yml

```diff
@@ -1,78 +1,75 @@
 modal:
-  label: "Modal"
-  description: "Use Bootstrap’s JavaScript modal plugin to add dialogs to your site for lightboxes, user notifications, or completely custom content. https://v5.getbootstrap.com/docs/5.0/components/modal/"
+  label: 'Modal'
+  description: 'Use Bootstrap’s JavaScript modal plugin to add dialogs to your site for lightboxes, user notifications, or completely custom content. For more information vitis https://v5.getbootstrap.com/docs/5.0/components/modal/'
   settings:
     btn_style:
       type: select
-      label: Button style
+      label: 'Button style'
+      description: 'Allows setting the style of the button:'
       options:
-        primary: primary
-        secondary: secondary
-        success: success
-        danger: danger
-        warning: warning
-        info: info
-        light: light
-        dark: dark
-        link: link
-      preview: primary
+        primary: 'primary (default)'
+        secondary: 'secondary'
+        success: 'success'
+        danger: 'danger'
+        warning: 'warning'
+        info: 'info'
+        light: 'light'
+        dark: 'dark'
+        link: 'link'
     size:
       type: select
-      label: Modal size
+      label: 'Modal size'
+      description: 'The size that the modal will have:'
       options:
-        sm: "sm"
-        lg: "lg"
-        xl: "xl"
-        fullscreen: "fullscreen"
-      preview: lg
-    modal_id:
-      type: textfield
-      label: "Modal id"
-      preview: ''
+        sm: 'sm'
+        lg: 'lg'
+        xl: 'xl'
+        fullscreen: 'fullscreen'
+      preview: 'lg'
   fields:
     btn_label:
       type: text
-      label: "Button label"
-      description: "The button text to trigger the modal. If is not present, then button will not be printed, only the modal container."
-      preview: "Open the modal"
+      label: 'Button label'
+      description: 'The button text to trigger the modal. If is not present, then button will not be printed, only the modal container.'
+      preview: 'Open the modal'
     header:
       type: text
-      label: "Header"
-      description: "Modal header"
+      label: 'Header'
+      description: 'The modal’s header.'
       preview:
```

Comment:
As I wrote for some other pattern, we should give an API. Instead of header, we should have "title" and wrap the title text in it. Same for button. @escuriola maybe remembers which other pattern had the same remark.